src/User.php

```diff
@@ -5965,7 +5965,7 @@ private function showLdapInformation(): void
                 $info = AuthLDAP::getUserByDn(
                     $ds,
                     $this->fields['user_dn'],
-                    ['*', 'createTimeStamp', 'modifyTimestamp']
+                    ['*', '+', 'createTimeStamp', 'modifyTimestamp']
```

Comment:
Could you explain and even give a link to a documentation that explains what is `+` means ?

Reply:
My bad, you already did.